practices/images-best-practices.md

```diff
@@ -0,0 +1,48 @@
+# Best practices when dealing with images for the Web
+
+In a front-end workflow we typically deal with two kinds of images:
+
+* **Icons, logos, buttons and other graphical elements** that are part of the User Interface of the page. These are usually managed by the frontend, and so they are stored with the rest of resources (templates, CSS, JS). Like those resources, they may be shared between many different pages.
+
+* **Content images**. These are unique to a page (or a handful of pages). They will be usually provided from a database or a content management system.
+
+Most best practices are common to both categories.
+
+## Formats
+
+### What format to use
+
+* Use SVGs whenever you want to display a vector image.
+* Use JPEGs for photo images, or images with many little details.
+* Use PNGs for graphs and other images with large areas of the same colour. Also for images that require transparency.
+* Don't use GIFs.
+
+### About WebP
+
+We don't encourage the use of the [WebP format](https://developers.google.com/speed/webp/) as support for it is still not widespread. [MS is not planning to add support to Edge](https://developer.microsoft.com/en-us/microsoft-edge/platform/status/webpimageformat/) at the moment. Firefox is currently working in adding support for it, so we may reconsider after that.
```

Comment:
Hmm not sure about this. It we didn't concatenate our JavaScript, we wouldn't write in the playbook "Don't concatenate your JavaScript". But it feels like that's what this text is doing?

If you purely want to state _What we currently do_ and exclude _What is a best practice applicable for everyone_, shouldn't we clearly differentiate between them? 

Also if you're discouraging it's use (which is how I read the text) - then I think it _is_ worth backing up in this PR. I like to share the frontend playbook with other members in the community, and at first glance, reading this document feels like we're recommending folks to _not_ use WebP - it's not entirely clear that you're simply stating, "we, Springer Nature, do not currently use WebP".